kubernetes/deployments_kubeclient.go

```diff
@@ -753,19 +763,19 @@ func (oc *openShiftAPIClient) GetBuildConfigs(namespace string, labelSelector st
 	return oc.getResource(bcURL, false)
 }
 
-func (kc *kubeClient) getEnvironmentNamespace(envName string) (string, error) {
+func (kc *kubeClient) getEnvironmentNamespace(envName string, includeInternal bool) (string, error) {
```

Comment:
A suggestion: have two methods, one that works as the old one did, and a new one, something like GetDeployableEnvrionmentNamespace(envName), and avoid the bool, which seems to be false most of the time anyways, and just clutters up the code.

Reply:
Sure, the boolean parameter made the code less readable.